docs/build/guides/transactions/pooled-accounts-muxed-accounts-memos.mdx

```diff
@@ -1,92 +1,549 @@
+Not all operations accept muxed addresses, but they can be used for:
+
+- The source account of any operation or transaction.
+- The fee source of a fee-bump transaction.
+- The destination of the three payment operations:
+  - `Payment`
+  - `PathPaymentStrictSend`
+  - `PathPaymentStrictReceive`
+- The destination of an `AccountMerge`.
+- The target (`from` field) of a `Clawback` operation.
+
+In practice, for exchanges and custodial services this covers:
 
-- The source account of any operation or transaction;
-- The fee source of a fee-bump transaction;
-- The destination of all three types of payments:
-  - `Payment`,
-  - `PathPaymentStrictSend`, and
-  - `PathPaymentStrictReceive`;
-- The destination of an AccountMerge; and
-- The target of a `Clawback` operation (i.e. the from field).
+- User deposits into a pooled account (M → G).
+- User withdrawals from the pooled account (G or M → external G or M).
+- Internal transfers between customers on the same pooled account (M → M).
+- Clawing back assets from a muxed customer balance (where the asset is clawback-enabled).
+- Contract token transfers where the exchange indexes deposits by muxed ID instead of memo ID.
 
-We will demonstrate some of these in the examples section.
+There is no validation on the ID at the protocol level. As far as the Stellar network is concerned, all supported operations behave exactly as if you had not used a muxed account; the ID is for your off-chain bookkeeping.
+
+### SDK integrations
+
+All official SDKs expose helpers for creating, parsing, and using muxed addresses. Here are a couple of minimal examples.
+
+#### JavaScript SDK:
+
+```js
+import * as sdk from "@stellar/stellar-sdk";
+
+const server = new sdk.rpc.Server("https://soroban-testnet.stellar.org");
+const custodianGAddress = "GA..."; // pooled account
 
-There’s no validation on IDs and as far as the Stellar network is concerned, all supported operations operate exactly as if you did not use a muxed account.
+// Load the underlying account once so we have its sequence number
+const custodianAccount = await server.getAccount(custodianGAddress);
+
+// Create a muxed account for customer ID "12345"
+const customerId = "12345"; // stringified uint64
+const muxed = new sdk.MuxedAccount(custodianAccount, customerId);
+
+console.log(muxed.accountId()); // prints an M-address like "MA7QY..."
+```
+
+Parsing an existing muxed address:
+
+```js
+const mAddress = muxed.accountId();
+
+// You need the base account's current sequence number to parse it
+const sequenceNum = custodianAccount.sequenceNumber();
+const parsed = sdk.MuxedAccount.fromAddress(mAddress, sequenceNum);
+
+console.log(parsed.baseAccount().accountId()); // underlying G-address
+console.log(parsed.id()); // "12345"
+```
+
+The same `MuxedAccount` object can be passed anywhere an account source or destination is expected when building transactions with `TransactionBuilder`.
+
+#### Elixir SDK:
```

Comment:
What's the reason for exampling with the Elixir SDK here? That SDK doesn't appear to have been updated since protocol 18. See https://hexdocs.pm/stellar_sdk/readme.html. It's probably not the best to use in examples. We probably only need one SDK as a general example, and folks really need to go to the SDKs to actually look at their docs.